db/trie/mptrie/branchnode.go

```diff
@@ -92,7 +94,7 @@ func newBranchNodeFromProtoPb(pb *triepb.BranchPb, hashVal []byte) *branchNode {
 	bnode.indices = NewSortedList(bnode.children)
 	bnode.cacheNode.serializable = bnode
 	if err := logNode(_nodeTypeBranch, _actionTypeNew, bnode, nil); err != nil {
-		panic(err)
```

Comment:
this is a fatal error which cannot be recovered unless retry. That's why it is a panic

Reply:
i%20have%20question%201.%20this%20is%20to%20log%20the%20node%20creation,%20why%20%22just%20print%20error%22%20would%20damage%20the%20db?%20i%20mean,%20before%20adding%20the%20log,%20this%20panic%20won%27t%20exist%202.%20in%20the%20log%20node%20commit,%20all%20other%20places%20are%20`return%20nil,%20err`,%20why%20only%20here%20need%20to%20panic?

Reply:
the log node commit: https://github.com/iotexproject/iotex-core/commit/dab026e7571a0770956f53f7c50f7896e246d723